nevergrad/optimization/oneshot.py

```diff
@@ -164,6 +178,7 @@ class RandomSearchMaker(base.ConfiguredOptimizer):
          - "random": uses a randomized pattern for the scale.
          - "auto": scales in function of dimension and budget (version 1)
          - "autolog": scales in function of dimension and budget (version 2)
+         - "autotune": scales in function of dimension and budget (version 3)
```

Comment:
at some point this will either require an equation for each and/or explanations on when to use each,
or deleting the ones that are not useful

Reply:
Equations added.
Some variants removed.